src/web/web-service/frontend/res/js/app.js

```diff
@@ -372,5 +381,404 @@
                 ev.target.disabled = false;
             });
         };
+
+
+        $scope.ipAddr = "0.0.0.0:8080";
```

Comment:
We are still hard-coding the IP address. https://stackoverflow.com/questions/1370745/how-to-find-out-the-server-ip-address-using-javascript-that-the-browser-is-con

Reply:
Yes, because location.host could get the IP address, but the port for restful API is different from the static server which for now need to be specified, so I just leave it here.

Reply:
I think we should use fix port number for restful API. Any disadvantage for of using fixed port number?

Reply:
Well, I think fix port number may bring inconvenience if port number conflicts with others, then one may need to go to source code level to fix it. Seem its better to specify it in a configure file which both agent and web could access, Is there such a file before?

Reply:
1. **port number**. I think there are many applications using fix port, for example, `adb` uses fixed 5555. The port number of the 
restful API should of cause be defined and fixed, otherwise. How can an user figure out the what the port numbe is without looking into the configuration number (a general user of cause has no access to configuration file)?

2. **ip address**. Does `0.0.0.0` points to the host server of the javascript file?

Reply:
I recommend we use `<otbr-agent-port> + 1` as the otbr-restful API port.